src/python/gem5/components/boards/arm_board.py

```diff
@@ -354,7 +354,7 @@ def _add_pci_device(self, pci_device: PciVirtIO) -> None:
         )
 
     @overrides(KernelDiskWorkload)
-    def get_disk_device(self):
+    def _get_default_disk_device(self):
```

Comment:
This is a virtual function, so i think, in this case, the method shouldn't lead with an underscore.

Reply:
Ditto for other instances in this patch where the leading underscore has been applied.